tests/clients/test_pymysql.py

```diff
@@ -243,3 +243,93 @@ def test_error_capture(self):
         self.assertEqual(db_span.data["mysql"]["stmt"], 'SELECT * from blah')
         self.assertEqual(db_span.data["mysql"]["host"], testenv['mysql_host'])
         self.assertEqual(db_span.data["mysql"]["port"], testenv['mysql_port'])
+
+    def test_connect_ctx_mngr(self):
+        result = None
+
+        with tracer.start_active_span('test'):
+            with self.db as conn:
+                cursor = conn.cursor()
+                result = cursor.execute("""SELECT * from users""")
+                cursor.fetchone()
+
+        assert(result >= 0)
```

Comment:
This `assert` is for catching runtime anomalies in the production code, after it is deployed.
For unit testing we should use [assertTrue](https://docs.python.org/3/library/unittest.html?highlight=assertisnone#unittest.TestCase.assertTrue), if nothing else fits better.